sda-svc/templates/ingest-deploy.yaml

```diff
@@ -60,86 +60,146 @@ spec:
-          value: "false"
-        - name: BROKER_VERIFY_PEER
-          value: {{ .Values.global.broker.verifyPeer | quote }}
-        - name: INBOX_STORAGE_DRIVER
-      {{- if eq "S3Storage" .Values.global.inbox.storageType }}
-          value: S3Storage
-        - name: INBOX_S3_URL
-          value: {{ required "S3 inbox URL missing" .Values.global.inbox.s3Url }}
-        - name: INBOX_S3_BUCKET
-          value: {{ required "S3 inbox bucket missing" .Values.global.inbox.s3Bucket }}
-        - name: INBOX_S3_REGION
-          value: {{ default "us-east-1" .Values.global.inbox.s3Region }}
-      {{- else }}
-          value: "FileStorage"
-        - name: INBOX_LOCATION
-          value: "{{ .Values.global.inbox.path }}/%s"
-      {{- end }}
-      {{- if not .Values.global.confFile }}
-        - name: BROKER_CONNECTION
+        {{- if not .Values.global.secretsService }}
+        - name: DB_PASSWORD
           valueFrom:
-            secretKeyRef:
-              name: {{ template "sda.fullname" . }}-ingest
-              key: mqConnection
-        - name: DB_CONNECTION
+              secretKeyRef:
+                name: {{ template "sda.fullname" . }}-ingest
+                key: dbPassword
+        - name: DB_USER
+          value: {{ include "dbUserIngest" . | quote }}
+        - name: BROKER_PASSWORD
           valueFrom:
-            secretKeyRef:
-              name: {{ template "sda.fullname" . }}-ingest
-              key: dbConnection
-      {{- if eq "S3Storage" .Values.global.archive.storageType }}
-        - name: ARCHIVE_S3_ACCESS_KEY
+              secretKeyRef:
+                name: {{ template "sda.fullname" . }}-ingest
+                key: mqPassword
+      {{- if eq "s3" .Values.global.archive.storageType }}
+        - name: ARCHIVE_ACCESSKEY
           valueFrom:
             secretKeyRef:
               name: {{ template "sda.fullname" . }}-s3archive-keys
               key: s3ArchiveAccessKey
-        - name: ARCHIVE_S3_SECRET_KEY
+        - name: ARCHIVE_SECRETKEY
           valueFrom:
             secretKeyRef:
               name: {{ template "sda.fullname" . }}-s3archive-keys
               key: s3ArchiveSecretKey
       {{- end }}
-      {{- if eq "S3Storage" .Values.global.inbox.storageType }}
-        - name: INBOX_S3_ACCESS_KEY
+      {{- if eq "s3" .Values.global.inbox.storageType }}
+        - name: INBOX_ACCESSKEY
           valueFrom:
             secretKeyRef:
               name: {{ template "sda.fullname" . }}-s3inbox-keys
               key: s3InboxAccessKey
-        - name: INBOX_S3_SECRET_KEY
+        - name: INBOX_SECRETKEY
           valueFrom:
             secretKeyRef:
               name: {{ template "sda.fullname" . }}-s3inbox-keys
               key: s3InboxSecretKey
+        - name: C4GH_PASSPHRASE
+          valueFrom:
+            secretKeyRef:
+              name: {{ template "sda.fullname" . }}-ingest
+              key: c4ghPassphrase
+      {{- end }}
+      {{- end }}
```

Comment:
Put this at the end like in verify and please order them alphabetically, it makes it easier to find the one you are looking for.